src/main/java/org/devcloud/ap/Azubiprojekt.java

```diff
@@ -9,24 +9,29 @@
 import org.devcloud.ap.utils.HTTPServer;
 import org.devcloud.ap.utils.apihelper.databsehelper.RoleDatabaseHelper;
 import org.devcloud.ap.utils.SentryLogger;
+import org.devcloud.ap.utils.Config;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 
 public class Azubiprojekt {
 
-    @Getter static SQLPostgres sqlPostgres;
+    @Getter
+    static SQLPostgres sqlPostgres;
     private static final Logger logger = LoggerFactory.getLogger(Azubiprojekt.class);
-    
+
     public static void main(String[] args) {
         SentryLogger.startSentry();
+        Config config = new Config();
```

Comment:
Besser in damit eine static variable zu erstellen um von außerhalb mit einer Methode zuzugreifen zu können.